src/components/files-upload-overview.less

```diff
@@ -0,0 +1,43 @@
+.FilesUploadOverview {
+  height: 100%;
+}
+
+.FilesUploadOverview-headline {
+  margin-bottom: 30px;
+}
+
+.FilesUploadOverview-usedStorage {
+  margin-bottom: 20px;
+}
+
+.FilesUploadOverview-fileStatus {
+  padding: 0 15px;
+  margin-bottom: 15px;
+}
+
+.FilesUploadOverview-fileStatusRow {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.FilesUploadOverview-fileStatusError {
+  padding-left: 25px;
+  font-size: 12px;
+  color: @edu-error-color;
+}
+
+.FilesUploadOverview-fileStatusContainer {
+  max-height: 70%;
+  overflow: scroll;
```

Comment:
I think this should be `auto`, as it otherwise always shows scrollbars (on non-mac machines ;-))

![localhost_3000_lessons_nKMNJJB1XE14DwvdcqmaTj_view=edit](https://user-images.githubusercontent.com/2676617/178539133-9768f5e3-36e6-4f6a-87b1-99fb275212c3.png)

Reply:
Good point 😳 Corrected!